README.md

```
@@ -197,3 +199,7 @@ This repo builds on work from:
 - [llama.cpp](https://github.com/ggml-org/llama.cpp)
 - [Lemonade](https://github.com/lemonade-sdk/lemonade)
 - Arch, CachyOS, and AUR package metadata used as package baselines
+
+
+## License
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)